components/Schedule/SportClassCard.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+export default function SportClassCard({ classData }) {
+  return (
+    <ClassCardContainer key={classData.id}>
```

Comment:
why is there a `key` here? This property is used for loops

Reply:
im using map in Schedule (line 41) and added the key. moving it to Schedule.